ICSharpCode.CodeConverter/CSharp/MethodBodyVisitor.cs

```diff
@@ -27,6 +27,7 @@ class MethodBodyVisitor : VBasic.VisualBasicSyntaxVisitor<SyntaxList<StatementSy
             private readonly SemanticModel _semanticModel;
             private readonly VBasic.VisualBasicSyntaxVisitor<CSharpSyntaxNode> _nodesVisitor;
             private readonly Stack<string> _withBlockTempVariableNames;
+            private readonly HashSet<string> _extraUsingDirectives;
```

Comment:
Nice, this is what I wanted to do, but I didn't see an existing way to do it.

Reply:
Yeah I was surprised I couldn't find it. Turns out it existed in just the C#->VB direction already.

Reply:
Ah, thinking about it, I think I had held off doing this previously due to paranoia about another using statement having conflicting imports. My ideal for this would probably be to add everything fully qualified *and* add the extra using statement. Then at the end do a "simplify names" analysis on the code to shorten names where they are unambiguous. If I could reference someone's existing roslyn fixer for that it'd be pretty easy I think.

Speaking of ambiguity, now I think about it, when creating new variables/methods, really ideally be using `GetUniqueVariableNameInScope` to avoid clashes.

Reply:
That occurred to me too, completely forgot to go back and fix that though. I'll bear it in mind.